index.html

```diff
@@ -253,11 +253,12 @@ <h3>Ecosystem Overview</h3>
 <a>verifiable credentials</a> for processing.
 Examples of verifiers include employers, security personnel, and websites.
         </dd>
-        <dt><a>identifier registry</a></dt>
+        <dt><a>Verifiable data repository</a></dt>
```

Comment:
This should be changed to something like "cryptographic material registry"

Reply:
I think schema is not a cryptographic material, neither are identifiers like a DID, do you agree? Can we call it **public data regsitry** so that it is clear that it contains no holder specific data

Reply:
Public data registry is too vague. Schema is cryptographic material -- it's used as input to cryptographic functions to perform calculations. Specifically, schemas enable byte streams to be created in a deterministic fashion, so yes, I would classify schemas as cryptographic material for the purposes of the VC Data Model.

Reply:
I am re-inclining to Verifiable Data Repository. Verifiable communicates the intent, cryptographic seems like a blanket term. As an example, seed to a PRF is a cryptographic data but not something we want to put in the repository